core/src/main/java/org/apache/calcite/sql/fun/SqlLibraryOperators.java

```diff
@@ -1123,6 +1123,16 @@ static RelDataType deriveTypeSplit(SqlOperatorBinding operatorBinding,
   public static final SqlSpecialOperator NOT_RLIKE =
       new SqlLikeOperator("NOT RLIKE", SqlKind.RLIKE, true, true);
 
+  @LibraryOperator(libraries = {MYSQL})
+  public static final SqlFunction BIT_COUNT_MYSQL =
```

Comment:
If only the name is different, you can try API use a copy of this function with a given name, like:
```
  /** Alias for {@link #BIT_GET}. */
  @LibraryOperator(libraries = {SPARK})
  public static final SqlFunction GETBIT =
      BIT_GET.withName("GETBIT");
```

Reply:
Thanks. There is now a difference in parameters now as well.